cloud_AWS/terraform/module/examples/all-vpc-from-region/README.md

```diff
@@ -21,14 +21,14 @@ Run the example:
 
 ```shell
 terraform init
-terraform apply
+terraform apply --vpc_id=<vpc-id>
```

Comment:
Is the --vpc_id a correct Terraform flag?

Reply:
Doesn't work with Terraform 1.0.4
Probably you meant --var vpc_id=<vpc-id>?
Note: recently the variable became a list of IDs: "vpc_id_list"

Reply:
Good catch. vpc_id is a TF variable defined in variables.tf for _single_vpc_ and _no_cloudexport_ example.
However there is no such variable in _all-vpc-from-region_ example so I have just removed that.

Reply:
BTW I added that, because passing variable in command is IMO more convenient than in interactive mode

Reply:
> Doesn't work with Terraform 1.0.4 Probably you meant --var vpc_id=? Note: recently the variable became a list of IDs: "vpc_id_list"

This particular example has no variables.